tests/PSinLinuxCloudShellImage.Tests.ps1

```diff
@@ -18,9 +18,8 @@ Describe "Various programs installed with expected versions" {
 
     It "Static Versions" {
         # These programs are installed explicitly with specific versions
-        $script:pmap["Node.JS"].Version | Should -Be '16.16.0'
+        $script:pmap["Node.JS"].Version | Should -Be '16.17.1'
         $script:pmap["PowerShell"].Version | Should -BeLike '7.2*'
-        $script:pmap["Helm"].Version | Should -BeLike '3.9.*'       
```

Comment:
Did we decide to remove helm from here? Not that I need it. Just curious.

Reply:
The reason we have nodejs and powershell version checking is because other packages depend on them. Helm isn't a package that other packages generally depend on. I think it'll be better to remove it for now - for consistency.